### PR TITLE
Fixes airlock cheese

### LINKED ISSUE #65620: When ID Scan wire is cut, doors can be opened by clicking but does not open when bumped
## Reproduction:
Cut any doors ID Scan wire. Use your hand to freely open the door. 
image
According to wiki it should prevent you from opening it by hand.

### PR DESCRIPTION
**Pull request by Colovorat**
## About The Pull Request

fixes: [65620](https://github.com/tgstation/tgstation/issues/65620)

## Why It's Good For The Game

it is cringe that cutting one wire allows everyone to open airlock
b3fd4215bea80088a083615aadf0e80c04673023

## Changelog

:cl:
fix: Clicking on airlock with ID Scan wire cut, should not open airlock.
/:cl:

### CODE REVIEW

**Comment by Colovorat** [comment]
> ok, aside from the balance, it still seems like this is not the right fix

as i said, it's just 6-12 years old code which i can't refactor, i just want to fix the issue

File: code/game/machinery/doors/door.dm [hunk 1]
```diff
@@ -210,8 +210,8 @@
 	add_fingerprint(user)
 	if(operating || (obj_flags & EMAGGED))
 		return
-	access_bypass |= !requiresID()
-	if(access_bypass || allowed(user))
+	access_bypass |= requiresID()
+	if(access_bypass && allowed(user))
```
Comment (Watermelon914) [inline code/game/machinery/doors/door.dm:214]: this is not a good solution and breaks more things, please change it around

**Review by Watermelon914 — CHANGES_REQUESTED** [review]

**Comment by Ghilker** [comment]
this is a balance change, it appears that the original coders intended to allow anyone to enter when the ID cable was disabled. This got changed 2 months ago for the bumping as an unrelated fix
image

**Comment by Colovorat** [comment]
> this is a balance change, it appears that the original coders intended to allow anyone to enter when the ID cable was disabled. This got changed 2 months ago for the bumping as an unrelated fix image

Statement from several maints said, that cutting wire, should deny access

**Comment by Ghilker** [comment]
i'm not saying you shouldn't do it, i'm saying that this is not a fix, is a balance change from how it operated before

**Comment by Likteer** [comment]
Will pulsing the open wire still work? I mentioned why here https://tgstation13.org/phpBB/viewtopic.php?f=9&t=31229

> You can no longer make airlocks opened by something else than ID reliably. The most painful loss here for me is making vault doors named SESAME /w voice analyzers

**Comment by Watermelon914** [comment]
> Will pulsing the open wire still work? I mentioned why here https://tgstation13.org/phpBB/viewtopic.php?f=9&t=31229
> 
> > You can no longer make airlocks opened by something else than ID reliably. The most painful loss here for me is making vault doors named SESAME /w voice analyzers

yes

**Comment by MacBlaze1** [comment]
> Will pulsing the open wire still work? I mentioned why here https://tgstation13.org/phpBB/viewtopic.php?f=9&t=31229
> 
> > You can no longer make airlocks opened by something else than ID reliably. The most painful loss here for me is making vault doors named SESAME /w voice analyzers

Im not sure the open wire works, Ive tried pulsing it on some doors and it does nothing

**Pull request merged**